src/full.js

```diff
@@ -35,7 +35,8 @@ const Components = {
 };
 
 if (typeof window !== 'undefined' && window.Vue) {
-  install(window.Vue);
+  const locale = global?.iso_user || "en";
```

Comment:
Why cannot use window instead of global object ? it seems to me that global was on the server side (nodejs) and window on the browser side, but effectively, global works also in browser.